src/capi-single-paid/index.js

```diff
@@ -65,4 +67,38 @@ function checkIcon(responseJson) {
     responseJson.videoTag ?
         videoIcon :
         '';
-};
+}
+
+function addSourceset(responseJson) {
+
+  let srcsetFragment = document.createDocumentFragment();
+
+  return responseJson.reduce((sources, source) => {
```

Comment:
Just to clarify, you're suggesting appending the highDef / lowDef elements to srcsetFragment?